### Hekateros hekateros_control/HekOpState message

int8   calib_state          # joint or robot calibration state

# calibration state enum
int8   UNCALIBRATED = 0     # uncalibrated
int8   CALIBRATING  = 1     # calibrating
int8   CALIBRATED   = 2     # calibrated

